### PR TITLE
 More attributes in Segy Standard Trace Header

### PR DESCRIPTION
**Pull request by devos0024**
Improve the parsing of Seg-y format according to seg_y_rev1 specification

### CODE REVIEW

**Comment by rouault** [comment]
cppcheck reports the following issue
```ogr/ogrsf_frmts/segy/ogrsegylayer.cpp:529,style,unreadVariable,Variable 'dfHorizontalScale' is assigned a value that is never used.
unreadVariable check failed```

File: gdal/ogr/ogrsf_frmts/segy/ogrsegylayer.cpp [hunk 1]
```diff
@@ -492,6 +526,8 @@ OGRFeature *OGRSEGYLayer::GetNextRawFeature()
     const double dfGroupX = nGroupX * dfHorizontalScale;
     const double dfGroupY = nGroupY * dfHorizontalScale;
 
+    dfHorizontalScale = nHorizontalScalar;
```
Comment (rouault) [inline gdal/ogr/ogrsf_frmts/segy/ogrsegylayer.cpp]: is this needed ?

Comment (devos0024) [reply gdal/ogr/ogrsf_frmts/segy/ogrsegylayer.cpp]: No. You are right, it's a mistake.

Comment (rouault) [reply gdal/ogr/ogrsf_frmts/segy/ogrsegylayer.cpp]: Can you push an additional commit to remove this ? thanks

**Pull request closed without merging**